dash/dash.py

```diff
@@ -1434,14 +1429,22 @@ def _validate_layout(self):
             component_ids.add(component_id)
 
     def _setup_server(self):
+        # Apply _force_eager_loading overrides from modules
+        eager_loading = self.config.eager_loading
+        for module_name in ComponentRegistry.registry:
+            module = sys.modules[module_name]
+            eager = getattr(module, '_force_eager_loading', False)
+            eager_loading = eager_loading or eager
+
+        # static files from the packages
+        self.css = Css(self.css.config.serve_locally)
+        self.scripts = Scripts(self.scripts.config.serve_locally, eager_loading)
```

Comment:
Do we need to recreate these here? Wouldn't it be sufficient to just say
```py
self.scripts.config.eager_loading = eager_loading
```
or even
```py
for module_name in ComponentRegistry.registry:
    module = sys.modules[module_name]
    if getattr(module, '_force_eager_loading', False):
        self.scripts.config.eager_loading = True
        break
```

Reply:
Ah! I think you're right. I think I misunderstood how/when this got evaluated. But the filter only gets called later.